pydeconz/models/sensor/air_quality.py

```diff
@@ -66,7 +68,7 @@ class AirQuality(SensorBase):
     @property
     def air_quality(self) -> str:  # AirQualityValue:
         """Air quality."""
-        return AirQualityValue(self.raw["state"]["airquality"]).value
+        return AirQualityValue(self.raw["state"].get("airquality")).value
```

Comment:
This will cause the unknown logging from the air quality enum.
```suggestion
        return AirQualityValue(self.raw["state"].get("airquality", "unknown")).value
```
I'm not really happy with my suggested change but, lets keep it for now

Reply:
merged suggestions